packages/edit-post/src/components/layout/index.js

```diff
@@ -377,9 +376,6 @@ function Layout( { initialPost } ) {
 								<MetaBoxes location="advanced" />
 							</div>
 						) }
-						{ isMobileViewport && sidebarIsOpened && (
-							<ScrollLock />
```

Comment:
Seems like component was added a long time ago for mobile but it's not present in the site editor and I'm not sure why we need it. Maybe @jasmussen or @ellatrix would know if this removal is impactful or not.